packages/components/nodes/vectorstores/Qdrant/Qdrant.ts

```diff
@@ -257,7 +265,10 @@ class Qdrant_VectorStores implements INode {
 
                     return res
                 } else {
-                    await QdrantVectorStore.fromDocuments(finalDocs, embeddings, dbConfig)
+                    for(let i=0; i<finalDocs.length; i+=batchSize) {
```

Comment:
if no `batchSize` is found, can we just do it without for loop?

I tried upserting 1k+ chunks, it works fine:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/b71a3038-5722-4a5f-820d-faf620a7d730)

Instead of doing this 10 times in for loop, I think its better for other to specify the batch size once they faced the error, if everything works fine for the first try, then thats great